resources/views/front/community-events/_event.blade.php

```diff
@@ -13,15 +13,14 @@
                 {{ $model->name }}
             </h3>
         </a>
-        @if ($model->jury_id)
-            <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }} - {!! __('front/community-events.fields.jury', ['user' => link_to_route('users.profile', $model->jury->name, [$model->jury])]) !!}</div>
-        @else
-            <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }}</div>
-        @endif
+            <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }}
+                @if ($model->jury_id)
+                 - {!! __('front/community-events.fields.jury', ['user' => link_to_route('users.profile', $model->jury->name, [$model->jury])]) !!}
```

Comment:
I forgot that users have two "names": one internal to their campaigns, and one for their public profiles. The correct one is accessed by `$model->jury->displayName()`